client/web/src/site-admin/WebhookCreateUpdatePage.tsx

```diff
@@ -213,17 +213,19 @@ export const WebhookCreateUpdatePage: FC<WebhookCreateUpdatePageProps> = ({ exis
                                 <Input
                                     className={classNames(styles.first, 'flex-1 mb-0')}
                                     message={
-                                        webhook.codeHostKind &&
-                                        webhook.codeHostKind === ExternalServiceKind.BITBUCKETCLOUD ? (
-                                            <small>Bitbucket Cloud doesn't support secrets.</small>
+                                        (webhook.codeHostKind &&
+                                            webhook.codeHostKind === ExternalServiceKind.BITBUCKETCLOUD) ||
+                                        webhook.codeHostKind === ExternalServiceKind.AZUREDEVOPS ? (
+                                            <small>Code Host doesn't support secrets.</small>
```

Comment:
```suggestion
                                            <small>Code host doesn't support secrets.</small>
```

(I also wonder whether we shouldn't have a small function, `supportsSecrets` somewhere? The check for BB Cloud and ADO is repeated 4 times on this page.